src/main/java/educationalproject/programmingstuff/controller/Controller.java

```diff
@@ -1,14 +1,57 @@
 package educationalproject.programmingstuff.controller;
 
+import educationalproject.programmingstuff.model.User;
+import educationalproject.programmingstuff.repository.ItemRepository;
+import educationalproject.programmingstuff.repository.OrderRepository;
+import educationalproject.programmingstuff.repository.UserRepository;
+import educationalproject.programmingstuff.service.TestService;
+import educationalproject.programmingstuff.service.UserServiceImpl;
+import lombok.RequiredArgsConstructor;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.ArrayList;
+import java.util.List;
+
 @RestController
+@RequiredArgsConstructor
 public class Controller {
 
-    @GetMapping("/test")
-    public String test() {
-        return "Don't bother me";
+    private final UserServiceImpl userService;
+
+    private final UserRepository userRepository;
+
+    private final ItemRepository itemRepository;
+
+    private final OrderRepository orderRepository;
+
+    private final TestService testService;
+
+    @GetMapping("/dataPrep")
+    public void dataPrep() {
+        testService.prepareTestData();
+    }
+
+    @GetMapping("/dataGet")
+    public List<List> dataGet() {
+
+        List<List> lists = new ArrayList<>();
+
+        lists.add(userRepository.findAll());
+        lists.add(itemRepository.findAll());
+        lists.add(orderRepository.findAll());
+
+        return lists;
+    }
+
+    @GetMapping("/userByName")
```

Comment:
According to the REST, we should name endpoints according to the domain entities, not by verbs or phrases. In this case it should be just "users", so that if you need all users, you should call it without any parameters, if you need a specific user and you know the id, you can call "users/123" and so on.
Our current endpoint controls the case, when you need all users, but with a filter by name.